dags/vz_cris_import.py

```diff
@@ -0,0 +1,39 @@
+import os
+import docker
+from datetime import datetime
+
+from airflow.decorators import dag, task
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.environ.get("ENVIRONMENT", 'development') # operating environment
```

Comment:
Should this be updated to be prod?

Reply:
or "production" i mean, I dont have a preference on the shorter version or spelled out

Reply:
Here 'development' is the default value that is assigned into the `DEPLOYMENT_ENVIRONMENT` variable when the containers environment doesn't provide anything. It will be set to production when the `ENVIRONMENT` variable contains that like on `atd-data03`. Our local stacks have it set to `development`, or if it's not defined, this line acts as a back-up.

Reply:
derp, yeah this is the default and not what its being set to. Totally forgot how this command worked